src/Identity/EntityFrameworkCore/src/RoleStore.cs

```diff
@@ -83,7 +83,7 @@ public class RoleStore<TRole, TContext, TKey, TUserRole, TRoleClaim> :
     /// </summary>
     /// <param name="context">The <see cref="DbContext"/>.</param>
     /// <param name="describer">The <see cref="IdentityErrorDescriber"/>.</param>
-    public RoleStore(TContext context, IdentityErrorDescriber describer = null)
+    public RoleStore(TContext context, IdentityErrorDescriber? describer = null)
```

Comment:
sorta random side question, are we going to attempt to switch to !! for things like context here eventually as well?

Reply:
I guess. Using `ArgNullException.ThrowIfNull` which is what `!!` would use is apparently ever so slightly faster, so it makes sense that way. That said, for a newly written code, I'm in favor of not having null checks and relying on the nullability annotations to provide guidance.